app/components/dem-info.jsx

```diff
@@ -0,0 +1,54 @@
+'use strict'
+import React from 'react';
+import cx from 'classnames';
+import insertSheet from 'react-jss';
+
+
+function DemInfo(props) {
+    const { user, classes, styles, className, ...otherProps } = props;
+
+    const userState = user?.state || 'N/A';
+    const userAge = user?.dob ? calculateAge(user.dob) : 'N/A';
+    const userPoliticalParty = user?.party || 'N/A';
+
```

Comment:
You brought up about the N/A.  After running the story I realize that the default should be an empty string, and if there are no parameters defined (eg user undefined) then the component should return an empty span - meaning not `|` visible.